index.html

```diff
@@ -9,11 +9,81 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+<nav class="top-navigation"> 
+    <div class="content">
+        <ul class="links">
+            <li class="active">Meet Karma</li>
+            <li>How it Works</li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+        </ul>
+        <img class="logo" src="https://thinkful-ed.github.io/karma-clone/img/karma-logo.svg" alt="Logo image" width="26">
+    </div>
+</nav>
+
 <body>
+    <header>
+        <section class="introduction">
+            <div class="content">
+            </div>
+            <h1>Introducing Karma</h1>
+            <h3>Bring WiFi with you, everywhere you go</h3>
```

Comment:
Hi Berry, very well done on your work, according to the light house reports of last week they said to me it is best practice to  have your H1 tags first then your H2 tags and then your H3 tags, but I could be wrong  and need to look at it myself again